<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerMainWindow</class>
 <widget class="QMainWindow" name="registerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
border-image: url(:/image-3/注册背景.jpg);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
#lineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background:  rgb(223,223,224);
    selection-background-color: darkgray;
}

</string>
    </property>
    <property name="placeholderText">
     <string>    username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
#lineEdit_2 {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background:  rgb(223,223,224);
    selection-background-color: darkgray;
}</string>
    </property>
    <property name="placeholderText">
     <string>    password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正少儿简体</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>REGISTER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
#lineEdit_3 {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background:  rgb(223,223,224);
    selection-background-color: darkgray;
}</string>
    </property>
    <property name="placeholderText">
     <string>confine password</string>
    </property>
   </widget>
   <widget class="QLabel" name="pixLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="informationLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文隶书</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>my personal information</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>132</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>男</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>女</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
#lineEdit_4 {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background:  rgb(223,223,224);
    selection-background-color: darkgray;
}</string>
    </property>
    <property name="placeholderText">
     <string>我的年龄：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>330</y>
      <width>331</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
#textEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background:  rgb(223,223,224);
    selection-background-color: darkgray;
}</string>
    </property>
    <property name="placeholderText">
     <string>我的个性签名：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
